src/pages/CatNew.js

```diff
@@ -28,21 +28,22 @@ export default class CatNew extends Component {
     }
 
     render() {
+        let createCat = this.state.newCat
```

Comment:
I would recommend object destructuring over creating a new variable. 
`let { newCat } = this.state`